profileViewer/modelViewer.js

```diff
@@ -21,6 +21,13 @@ function addTouchDots({ motionController, rootNode }) {
     if (component.dataSource.dataSourceType === 'touchpadSource') {
       // Find the node to attach the touch dot.
       const componentRoot = rootNode.getObjectByName(component.rootNodeName, true);
+
+      if (!componentRoot) {
+        // eslint-disable-next-line no-console
+        console.error(`Could not find root node of touchpad component ${component.rootNodeName}`);
```

Comment:
Can we set this text in the assetErrorElement too?  Would make it easier for artists.  Same in the other places you're logging errors.  I'd be happy for this to be a helper function too.